### ISSUE TITLE
Define a smallest quadrilateral shape

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/quadrilateral#19]
Discussed 10/26/21 - At the moment we want to make it so that the vertices don't overlap and never obscure each other. This will simplify mouse/touch input by making it so that it is clear which vertex will receive input. It was proposed that the largest pointer area (mouseArea or touchArea) will be used to define the smallest shape.

**Comment by jessegreenberg** [comment]
This is made a bit more complicated by #15. Instead of rectangular bounds with limits that can be defined from the pointer areas, the available space for each vertex can be an arbitrary shape.

**Comment by jessegreenberg** [comment]
I guess instead of this requirement changing the available shape we could just add a separate constraint that none of the vertices can overlap. I think to accomplish this we should add a vertex size in model space. This will inform the overlap which will be calculated by the model. The touch areas can then be set from the size in view space.

**Comment by BLFiedler** [comment]
We touched on this in Virtual Model meeting and TS and BF agreed we could make a vertex size in the model that the touch area could use for its parameter.

Noting here that from my understanding, cursor area will be smaller. May be its own thing or a scaled value of the vertex size?

**Comment by jessegreenberg** [comment]
This was done in the above commits. The vertices cannot overlap each other. Adding label to review with the team.
1) Size of the vertices reasonable?
2) Should vertices be able to intersect with sides?

**Comment by BLFiedler** [comment]
> This was done in the above commits. The vertices cannot overlap each other. Adding label to review with the team.
> 
> 1. Size of the vertices reasonable?
> 2. Should vertices be able to intersect with sides?

1.) Fine for larger displays right now - up for discussion later on mobile devices. Will create a new issue.
2.) Addressed in #15
